presto-jdbc/src/main/java/io/prestosql/jdbc/ConnectionProperties.java

```diff
@@ -41,6 +41,7 @@
     public static final ConnectionProperty<HostAndPort> SOCKS_PROXY = new SocksProxy();
     public static final ConnectionProperty<HostAndPort> HTTP_PROXY = new HttpProxy();
     public static final ConnectionProperty<String> APPLICATION_NAME_PREFIX = new ApplicationNamePrefix();
+    public static final ConnectionProperty<String> CLIENT_TAGS = new ClientTags();
```

Comment:
Move this to be below `EXTRA_CREDENTIALS`

Add it to the `ALL_PROPERTIES` list